pep-0001.txt

```diff
@@ -136,35 +135,46 @@ python-ideas.  This gives the author a chance to flesh out the draft
 PEP to make properly formatted, of high quality, and to address
 initial concerns about the proposal.
 
-Following a discussion on python-ideas, the proposal should be sent as a
-draft PEP to the PEP editors <peps@python.org>.  The draft must be written
-in PEP style as described below, else it will be sent back without further
-regard until proper formatting rules are followed (although minor errors
-will be corrected by the editors).
-
-If the PEP editors approve, they will assign the PEP a number, label it
-as Standards Track, Informational, or Process, give it status "Draft",
-and create and check-in the initial draft of the PEP.  The PEP editors
-will not unreasonably deny a PEP.  Reasons for denying PEP status
-include duplication of effort, being technically unsound, not
-providing proper motivation or addressing backwards compatibility, or
-not in keeping with the Python philosophy.  The BDFL can be consulted
-during the approval phase, and is the final arbiter of the draft's
-PEP-ability.
-
-Developers with git push privileges for the `PEP repository`_ may claim
-PEP numbers directly by creating and committing a new PEP. When doing so,
-the developer must handle the tasks that would normally be taken care of by
-the PEP editors (see `PEP Editor Responsibilities & Workflow`_). This
-includes ensuring the initial version meets the expected standards for
-submitting a PEP. Alternately, even developers may choose to submit PEPs
-through the PEP editors. When doing so, let the PEP editors know you have
-git push privileges and they can guide you through the process of updating
-the PEP repository directly.
+
+Submitting a PEP
+----------------
+
+Following a discussion on python-ideas, the proposal should be submitted as a
+draft PEP via a `GitHub pull request`_.  The draft must be written in PEP
+style as described below, else it will fail review immediately (although minor
+errors may be corrected by the editors).
+
+The standard PEP workflow is:
+
+* You, the PEP author, clones the `PEP repository`_, and create a file named
+  `pep-xxxx.rst` that contains your new PEP.  Use "XXX" as your draft PEP
```

Comment:
I don't want to block this because of python.org. I'd say keep it as-is. I've already wrote the code so I just need a couple of hours to write a unit test and cleanup the code :)